### PR TITLE
Add v1.2.1 of mattermost-plugin-mscalendar to the Marketplace

### PR DESCRIPTION
**Pull request by mickmister**
#### Ticket Link

https://github.com/mattermost/mattermost-plugin-mscalendar/pull/255

### CODE REVIEW

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested upgrade path
- Tested fresh install path
- No changes to icon
- As a precaution - Ensured the plugin could start on a cloud server and accept commands
- No changes to icon 

LGTM!

**Review by neallred — APPROVED** [review]
LGTM, thanks @mickmister ! Validated that the only changes were signatures and version numbers.

File: plugins.json [hunk 1]
```diff
@@ -7542,6 +7542,134 @@
...
+            "placeholder": "",
+            "default": "",
+            "hosting": ""
+          }
+        ]
+      }
+    },
+    "platforms": {
+      "linux-amd64": {
+        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-mscalendar-v1.2.1-linux-amd64.tar.gz",
```
Comment (trilopin) [inline plugins.json:7659]: are these plugins-store.test.mattermost URLs valid? The "test" part confuses me a bit.

**Review by trilopin — APPROVED** [review]

**Pull request merged**